spec/routes_spec.rb

```diff
@@ -17,6 +19,41 @@
           it 'method defined' do
             expect(subject.respond_to?(action)).to be_truthy
           end
+
+          describe 'RBAC' do
+            before(:all) { MiqProductFeature.seed_features }
```

Comment:
Not sure if this will have a problem with CI...I know we've seen issues in the past due to the way it doesn't clean up after itself.  @jrafanie Thoughts?

Reply:
Actually now that I'm looking closer, there is only one `it` in this describe, so this can just be a normal `before`.

If you _want_ to use `before(:all)` it probably makes more sense up between lines 14 and 15

Reply:
Yup, gonna do that :+1:

Reply:
before(:all) is only ok if the tests are expected to work with all of the features.  If there are tests that need to test only a subset of features or assume certain numbers of features or order of features, they can break when new features are added or removed.

I'm ok with `before(:all)` where it makes sense but that intention should be clear and only affect a single file, so there should be `after(:all)` that cleans up afterwards since it's hard to demonstrate cross-file assumptions/intent in regards to seeded data.

Reply:
How this cleanup should look like? FYI this spec runs as a separate job called [`spec:routes`](https://travis-ci.com/github/ManageIQ/manageiq-ui-classic/jobs/469551522) so it doesn't affect our other specs in the default build job, so we might don't need to clean up at all :thinking: